gateway-server/src/main/java/org/apache/knox/gateway/GatewayServer.java

```diff
@@ -923,6 +925,40 @@ private synchronized void internalActivateArchive( Topology topology, File warDi
     }
   }
 
+  private synchronized void activateHomeApp() {
+    try {
+      // UI
+      activateHomepageContext("/home", "homepage/%2Fhome");
```

Comment:
why the weird `%2F` - which is a url encoded forward slash...

Reply:
I've just followed the process which is already in place for dynamic topology deployment. That process also creates `%2F` and `%2Fadmin-ui` folders where necessary. When tested, it was not even working w/o these prefixes.